tests/test_permissions.py

```diff
@@ -257,20 +259,22 @@ def test_permissions_model_process(
     )
 
     spec = factory.create_traintuple(
-        algo=algo_2, inputs=dataset_2.train_data_inputs + FLTaskInputGenerator.trains_to_train([traintuple_1.key])
+        algo=algo_2,
+        inputs=dataset_2.train_data_inputs
+        + FLTaskInputGenerator.trains_to_train([traintuple_1.key], worker=workers[1]),
```

Comment:
Why adding worker arg as the method doesn't expect that and the input don't need this value?

Reply:
OK I think I understand: the worker arg is for `create_traintuple` not `trains_to_train`